packages/wrangler/src/__tests__/config/configuration.test.ts

```diff
@@ -6396,6 +6396,117 @@ describe("normalizeAndValidateConfig()", () => {
 				expect(result2.diagnostics.hasWarnings()).toBe(false);
 			});
 		});
+
+		describe("[assets]", () => {
+			it("should inherit from top-level assets", () => {
```

Comment:
I feel like we should be already having a test for this, but then again here we are...so just to be abs sure, can we test that if we config assets at both top level and env level, the config resolves correctly?

Reply:
Good call, added